libs/common/src/assets/i18n/en.json

```diff
@@ -983,6 +984,153 @@
+	"tasks-automation.badge.beta": "Beta",
+	"tasks-automation.badge.optional": "Optional",
+	"tasks-automation.cta.new-task": "New task",
+	"tasks-automation.cta.see-archive": "See Archive",
+	"tasks-automation.errors.counting-labeled-resources": "An error occurred while loading the number of resources labeled with selected label set.",
+	"tasks-automation.errors.counting-resources": "An error occurred while loading the number of resources matching the task configuration.",
+	"tasks-automation.filters.all-resources": "All resources",
+	"tasks-automation.form.apply-task-to.automation": "Apply automatically to <strong>all existing and future resources</strong>",
+	"tasks-automation.form.apply-task-to.description": "Keep the task running for future resources or run it once.",
+	"tasks-automation.form.apply-task-to.one-time": "Run once on <strong>existing resources only</strong>",
+	"tasks-automation.form.apply-task-to.title": "What do you want the task to apply to?",
+	"tasks-automation.form.field-name.label": "Name the field",
+	"tasks-automation.form.field-name.placeholder": "Field name",
+	"tasks-automation.form.filter.all": "all",
+	"tasks-automation.form.filter.description": "The action will apply only to resources matching the selection.",
+	"tasks-automation.form.filter.languages.all-languages": "All languages",
+	"tasks-automation.form.filter.languages.button-label": "Languages",
+	"tasks-automation.form.filter.resource-labels.button-label": "Resource labels",
+	"tasks-automation.form.filter.resource-type.all-types": "All resource types",
+	"tasks-automation.form.filter.resource-type.button-label": "Resource types",
+	"tasks-automation.form.filter.search-in.content": "Content contains",
+	"tasks-automation.form.filter.search-in.title": "Title contains",
+	"tasks-automation.form.filter.search-in.title-or-content": "Title or content contains",
+	"tasks-automation.form.filter.search-input-label": "Search",
+	"tasks-automation.form.filter.title": "Filter the resources",
+	"tasks-automation.form.label-set-selection.create-label-set-button": "Create new label set",
+	"tasks-automation.form.label-set-selection.description-resources": "Select the label set that you want to use to label resources.",
+	"tasks-automation.form.label-set-selection.description-text-blocks": "Select the label set that you want to use to label text blocks.",
+	"tasks-automation.form.label-set-selection.resources-with-labels": "<strong>{{count}} resources</strong> are labeled with the labels from this label set.",
+	"tasks-automation.form.label-set-selection.select-dropdown-label": "Select label set",
+	"tasks-automation.form.label-set-selection.text-blocks-with-labels": "<strong>{{count}} text blocks</strong> are labeled with the labels from this label set.",
+	"tasks-automation.form.label-set-selection.title": "Select label set",
+	"tasks-automation.form.labeling-options.description-resources": "Specify the source for the synthetic labeling of your resources.",
```

Comment:
is `synthetic labeling` an official term? (I do not understand what it means)

Reply:
I don't really know neither, I got this from the design. @vincentgarnier or Anna should be able to answer better than me